### PR TITLE
Fix code format

### PR DESCRIPTION
**Pull request by lindexi**
## Summary

Fix code format and fix the API usage.

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]

**Comment by IEvangelist** [comment]
Hi @lindexi - thank you for fixing this. There is a build error, could you please also add a new .csproj for this?

>  Check failure on line 1 in samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/example.cs
> 
> GitHub Actions
> / snippets-build
> 
> samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/example.cs#L1
> 
> 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code

**Comment by lindexi** [comment]
@IEvangelist  Sorry, I can not compile the project after I added the csproj file. Because we define the same MainWindow class in `example.cs` and `example3.cs` and `example4.cs`

Should I rename the MainWindow to MainWindow3 or MainWindow4 in  `example3.cs` and `example4.cs` ?

File: samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0-windows</TargetFramework>
+    <Nullable>enable</Nullable>
+    <UseWPF>true</UseWPF>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```
Comment (Youssef1313) [inline samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj:7]: Why is this needed?

Comment (lindexi) [reply samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj:7]: @Youssef1313  Because we define the same MainWindow class in example.cs and example3.cs and example4.cs

Comment (Youssef1313) [reply samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj:7]: The problem with `EnableDefaultCompileItems` being false is that it beats the purpose of the csproj, which is to make sure the code compiles properly. If the examples are completely independent, I'd make a folder for each of them, each with its own csproj. @IEvangelist Do you agree?

Comment (Youssef1313) [reply samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj:7]: Actually `example3` and `example4` doesn't seem to be referenced. So they can just be deleted?

Comment (lindexi) [reply samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj:7]: @IEvangelist Can I delete the example3.cs and example4.cs file and create the MainWindow2.xaml file?

Comment (IEvangelist) [reply samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj:7]: You'd have to verify that those two files are in fact not being referenced, if you can prove that then yes.
